### ISSUE TITLE
How to build this on Qt

### ISSUE THREAD

**Issue by reanaagrawal** [chrizbee/NewtonFractal#12]
Hey, first of all, thank you so much for publishing this code. I'm very new to this so I'm confused and need a little help. How exactly are we supposed to build this with qmake (on Mac)…I tried cloning the repo with Xcode and also cloning the repo in Qt creator but somehow I am not able to run the application. Can you please explain to me this? I need to create a software tool for Newton Fractals and other equational visualisations for my college project. I'll be really really thankful if you could help me.

**Comment by chrizbee** [comment]
Sorry for my late response - I hope you figured it out already.

You don't actually have to use Qt Creator or XCode to get and build this repository. You can use the commands described [here](https://github.com/chrizbee/NewtonFractal#building) with only your commandline. Make sure to install Qt5 before!

TL;DR
```
git clone https://github.com/chrizbee/NewtonFractal
cd NewtonFractal
qmake
make
build/NewtonFractal
```
